From 1b0e574d680101105a6c1e8931c78824f5a97a42 Mon Sep 17 00:00:00 2001
From: Otavio Salvador <otavio@ossystems.com.br>
Date: Mon, 5 Sep 2016 17:08:56 +0000
Subject: [PATCH] Makefile: Do not fail if CONFIG_TRACEPOINTS is not enabled
Organization: O.S. Systems Software LTDA.

The lttng-modules are being pulled by the tools-profile image feature,
however, not every kernel has the CONFIG_TRACEPOINTS feature enabled.

This change makes the build do not fail when CONFIG_TRACEPOINTS is not
available, allowing it to be kept being pulled by default.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>
[bva: modified for lttng-modules 2.13+ context]
Signed-off-by: Bruce Ashfield <bruce.ashfield@gmail.com>
---
 Makefile | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

Index: lttng-modules-2.13.0/Makefile
===================================================================
--- lttng-modules-2.13.0.orig/Makefile
+++ lttng-modules-2.13.0/Makefile
@@ -7,6 +7,11 @@
 
 obj-$(CONFIG_LTTNG) += src/
 
+  ifdef CONFIG_LOCALVERSION	# Check if dot-config is included.
+    ifneq ($(CONFIG_TRACEPOINTS),)
+      $(warning The option CONFIG_TRACEPOINTS needs to be enabled in your kernel configuration)
+    endif # CONFIG_TRACEPOINTS
+  endif # ifdef CONFIG_LOCALVERSION
 else # KERNELRELEASE
 
 # This part of the Makefile is used when the 'make' command is runned in the
